pkg/kwrapper/k8s/k3s_linux.go

```diff
@@ -41,15 +34,17 @@ func getEmbedded(ctx context.Context) (bool, clientcmd.ClientConfig, error) {
 	return true, clientConfig, nil
 }
 
-func k3sServer(ctx context.Context, endpoints []string) (string, error) {
+func k3sServer(ctx context.Context) (string, error) {
+	// --cluster-init will start k3s with embedded etcd
 	cmd := exec.Command("k3s", "server",
+		"--cluster-init",
 		"--no-deploy=traefik",
 		"--no-deploy=servicelb",
 		"--no-deploy=metrics-server",
 		"--no-deploy=local-storage",
```

Comment:
The no-deploy flag is deprecated, can you replace this with:
```suggestion
		"--disable=traefik,servicelb,metrics-server,local-storage",
```

Reply:
Yep glad I brought that up! Will do